src/geo/transform.js

```diff
@@ -560,6 +565,10 @@ class Transform {
         mat4.rotateZ(m, m, this.angle);
         mat4.translate(m, m, [-x, -y, 0]);
 
+        // The mercatorMatrix can be used to transform points from mercator coordinates
+        // ([0, 0] nw, [1, 1] se) to GL coordinates.
+        this.mercatorMatrix = mat4.scale([], m, [this.worldSize, this.worldSize, this.worldSize]);
```

Comment:
Any reason we're using regular JS array here for the matrix instead of typed one like in other places, e.g. `new Float64Array(16)`?

Reply:
Yep, the gl uniform commands (`uniformMatrix4fv(...)`) don't accept `Float64Array`. Using them would force the user to convert them into a regular array or `Float32Array`. I'm not sure there would be any advantage to using `Float64Array`s instead of regular arrays.

We should review whether there is a good reason to be using Float64Array elsewhere. I know we've been doing it for a long time but I'm not sure if there is an actual advantage.